mgmt/imgmgr/src/imgmgr.c

```diff
@@ -28,12 +28,14 @@
 #include "bootutil/image.h"
 #include "bootutil/bootutil.h"
 #include "mgmt/mgmt.h"
+#include "log/log_fcb_slot1.h"
```

Comment:
I recommend making these #include lines to be behind #ifdef MYNEWT(LOG_FCB_SLOT1), as sys/log/<xxx> might not be visible to mgmt/imgmgr package.

Also, I think you're missing a change to pkg.yml, which conditionally includes sys/log/full iff LOG_FCB_SLOT1 is set. This should be full name of the package, not the 'log' api, as you have the API definition header file in sys/log/full only.

Reply:
I added slot1 APIs to `sys/log/stub` and added `pkg.req_apis` here
this also allows to build app which uses slot1 with `sys/log/stub' if necessary